examples/simple-form/index.jsx

```diff
@@ -55,10 +48,7 @@ const checkboxProps = {
   ],
 };
 
-// eslint-disable-next-line @typescript-eslint/no-unsafe-call
-void defineCustomElements();
-
-const App = () => (
+const SimpleApp = () => (
```

Comment:
I think this accidentally got reverted, but it shouldn't matter too much. My plan is to submit a PR that updates the simple example to use the form I demoed at the end of sprint demo on 4/27.